internal/store/pod.go

```diff
@@ -580,6 +583,37 @@ func createPodInfoFamilyGenerator() generator.FamilyGenerator {
 	)
 }
 
+func createPodIPFamilyGenerator() generator.FamilyGenerator {
+	return *generator.NewFamilyGenerator(
+		"kube_pod_ips",
+		"Pod IP addresses",
+		metric.Gauge,
+		"",
+		wrapPodFunc(func(p *v1.Pod) *metric.Family {
+			ms := make([]*metric.Metric, len(p.Status.PodIPs))
+			labelKeys := []string{"ip", "ip_family"}
+
+			for i, ip := range p.Status.PodIPs {
+				netIP := net.ParseIPSloppy(ip.IP)
```

Comment:
> why did you prefer `ParseIPSloppy` over `ParseIP` here?

I was trying to avoid having to munge import names -- `k8s.io/utils/net` contains `ParseIPSloppy` as well as the `IsIPv4` function. I don't have a strong opinion here though, so if you'd like to see `ParseIP`, I'm happy to make that change.

> Should we handle any errors (e.g. if ip.IP is not an actual IP it will return nil)?

How reliable do we consider the IPs from pod status? I can check for nil and not include it in the returned metric family.

Reply:
Thanks for the clarification. Let's stick with ParseIPSloppy then and not include in the metric output if ParseIPSloppy returns nil.